tests/NgTests/DnxCatalogCollectorTests.cs

```diff
@@ -88,5 +91,46 @@ public class DnxCatalogCollectorTests
             Assert.Null(otherPackageNuspec.Key);
             Assert.Null(otherPackageNupkg.Key);
         }
+
+        [Theory]
+        [InlineData("1.2.0")]
+        [InlineData("1.2")]
+        [InlineData("0.1.2")]
+        [InlineData("1.2.3.0")]
+        [InlineData("1.2.3.4")]
+        [InlineData("1.2.3-beta1")]
+        [Description("Test the dnxmarker save and delete scenarios.")]
+        public async Task DnxMarkerTestVersion(string version)
+        {
+            string id = "testid";
+            // Arrange
+            var catalogToDnxStorage = new MemoryStorage();
+            var catalogToDnxStorageFactory = new TestStorageFactory(name => catalogToDnxStorage.WithName(name));
+            DnxMaker marker = new DnxMaker(catalogToDnxStorageFactory);
+
+            var nupkg = new MemoryStream();
+            StreamWriter writer = new StreamWriter(nupkg);
+            writer.Write("nupkg data");
+            writer.Flush();
+            nupkg.Position = 0;
+
+            //Act
```

Comment:
please add spaces between the `//` and the comment itself

e.g. `// Act`

Also, when I have multiple `Act`s and `Assert`s, I usually number them. I think it looks nice but you don't have to.

e.g.
```
// Act 1
...
// Assert 1
...
// Act 2
...
// Assert 2
...
```